pkgs/development/python-modules/cli-helpers/default.nix

```diff
@@ -19,12 +21,13 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [
+    configobj
     terminaltables
     tabulate
     wcwidth
```

Comment:
This seems to no longer be a dependency.

Reply:
This was the initial failure as I have indicated in motivation.

Even in master (e.g. 3d5025ba79d) there is currently a reproducible failure:

```
git clone https://github.com/NixOS/nixpkgs
cd nixpkgs
nix-build -E 'with import ./. { }; pgcli'
```

Reply:
@dotlambda it's in `install_requires`

Reply:
I meant `wcwidth`. I can't find `wcwidth` in https://github.com/dbcli/cli_helpers/blob/v1.1.0/setup.py#L41-L45.

Reply:
@dotlambda sorry, I thought you were commenting the change, not just the last line. 

I have tried removing `wcwidth` and I get the following error:

```
Collecting wcwidth; extra == "widechars" (from tabulate[widechars]>=0.8.2->cli-helpers==1.1.0)
  Could not find a version that satisfies the requirement wcwidth; extra == "widechars" (from tabulate[widechars]>=0.8.2->cli-helpers==1.1.0) (from versions: )
No matching distribution found for wcwidth; extra == "widechars" (from tabulate[widechars]>=0.8.2->cli-helpers==1.1.0)
builder for '/nix/store/18wzsymnny12jm8vyc930fhbzkpr7qg8-python3.7-cli_helpers-1.1.0.drv' failed with exit code 1
```

Seems to me as a propagated dependency.

Reply:
I see, then it's good.